src/components/LooMap/LooMap.js

```diff
@@ -85,12 +93,34 @@ const LooMap = ({
               push(`/loos/${toilet.id}`);
             }
           }}
-          keyboard={!staticMap}
+          keyboard={false}
         />
       )),
     [loos, staticMap, push]
   );
 
+  if (withAccessibilityOverlays && mapRef.current) {
+    const map = mapRef.current.leafletElement.getContainer();
+
+    const callback = function (mutationsList) {
+      for (let mutation of mutationsList) {
+        const focusVisible = mutation.target.dataset.focusVisibleAdded === '';
+
+        if (focusVisible !== renderAccessibilityOverlays) {
+          setRenderAccessibilityOverlays(focusVisible);
+        }
+      }
+    };
+
+    const observer = new MutationObserver(callback);
+
+    // only render accessibility overlays when [data="focus-visible-added"] is applied
+    //
+    // we conditionally render instead of toggling CSS display since we want to avoid AccessibilityList being announced
+    // before the map is keyboard focused
+    observer.observe(map, { attributes: true });
+  }
```

Comment:
Creating a mutation observer is a side effect of the component so should probably go inside a useEffect with a disconnect cleanup